.github/workflows/aa_cc_kbc.yml

```diff
@@ -71,6 +71,7 @@ jobs:
 
       - name: Build and install with ${{ matrix.attester }} feature
         run: |
+          sudo apt-get update
```

Comment:
How about making it a step of its own before tpm deps install (and move apt update from that step) so that apt update is run only once

Reply:
Yes, let me put another PR to apply this idea. and get this merged to fix.